src/cli.ts

```diff
@@ -2,6 +2,16 @@ import readline from "readline";
 import { Agent } from "./evo";
 import { FileSystemWorkspace } from "./workspaces";
 
+const rl = readline.createInterface({
```

Comment:
I think this is cleaner on the bottom.
When someone opens the file they don't care about rl or prompt. They want to see the CLI logic.